Makefile

```diff
@@ -42,7 +42,7 @@ dependencies: carthage-bootstrap configs secrets
 bootstrap: hooks dependencies
 
 carthage-bootstrap:
-	bin/carthage.sh || exit 1
+	bin/carthage.sh bootstrap --platform iOS || exit 1
```

Comment:
@jgsamudio catching up here - strange, this should have been handled within the script itself based on whether it's running on CI or not 🤔

Reply:
Yeah I was running into issues with this script where I needed to specify bootstrap and the platform. I agree that its odd that it didn't run the command for local builds